contrib/confluentinc/confluent-kafka-go/kafka.v2/option.go

```diff
@@ -104,3 +107,19 @@ func WithCustomTag(tag string, tagFn func(msg *kafka.Message) interface{}) Optio
 		cfg.tagFns[tag] = tagFn
 	}
 }
+
+// WithConfig extracts the config information for the client to be tagged
+func WithConfig(cg *kafka.ConfigMap) Option {
+	return func(cfg *config) {
+		if bs, err := cg.Get("bootstrap.servers", ""); err == nil && bs != "" {
+			var hostnames []string
+			for _, addr := range strings.Split(bs.(string), ",") {
+				host, _, err := net.SplitHostPort(addr)
+				if err == nil {
+					hostnames = append(hostnames, host)
+				}
+			}
+			cfg.bootstrapServers = hostnames[0]
```

Comment:
Do we need to build the whole hostname list if we only care about the first one?
```suggestion
			for _, addr := range strings.Split(bs.(string), ",") {
				host, _, err := net.SplitHostPort(addr)
				if err == nil {
					cfg.bootstrapServers = host
					return
				}
			}
```

Or is this supposed to capture all of them? It seems a little strange to pick just one out of the list, since none is more important than any other for `bootstrap.servers`.

Reply:
Good idea, we only need the first hostname anyway because we dont want a list of servers